### PR TITLE
support to update existing report

### PR DESCRIPTION
**Pull request by memodi**
- updates existing report - helps if you're looking at multiple runs with different nightly builds, so one could have unified JSON file with failures across all runs on the day they're doing CI Watching.
- add polarion run id and nightly version information along with logs and profile.

### CODE REVIEW

**Comment by memodi** [comment]
@jogeo - could you help review this PR.

File: parse_ci_monitor_json.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 import subprocess
 import pprint
 from datetime import datetime
-from typing import List
+from typing import List, Set
 
 
 def argparser():
```
Comment (nathan-weinberg) [inline parse_ci_monitor_json.py:14]: There's a pretty good `argparse` library for Python if you're interested: https://docs.python.org/3/library/argparse.html

**Review by nathan-weinberg — APPROVED** [review]
LGTM

File: parse_ci_monitor_json.py (hunk 1 above)
Comment (memodi) [reply parse_ci_monitor_json.py:14]: I actually use that same library. Perhaps not clear here if you just look at the changes in this commit.

**Comment by jogeo** [comment]
I tried this out with several Run IDs that I know had the same failures. Worked well to group those failures.
LGTM

**Review by jogeo — APPROVED** [review]

**Pull request merged**